packages/controls/src/widget_image.ts

```diff
@@ -23,6 +23,13 @@ class ImageModel extends CoreDOMWidgetModel {
             value: new Uint8Array(0)
         });
     }
+
+    static serializers = {
+        ...CoreDOMWidgetModel.serializers,
+        value: {serialize: (value, manager) => {
+            return new DataView(value instanceof ArrayBuffer ? value : value.buffer);
```

Comment:
Since we technically need a copy, wouldn't it be better to do a slice? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice

Reply:
Fixed